pkgs/data/fonts/generic/default.nix

```diff
@@ -0,0 +1,42 @@
+{ pkgs, lib, stdenv, callPackage }:
+
+{ buildInputs ? [], nativeBuildInputs ? []
+, enableParallelBuilding ? true
+, meta ? {}
+, platforms ? lib.platforms.all
+, ... } @ args:
+
+stdenv.mkDerivation ( args // {
+  pname = "${lib.getName args}";
+  version = lib.getVersion args;
+
+  buildInputs = buildInputs;
+  nativeBuildInputs = nativeBuildInputs;
```

Comment:
Why not just remove this? It would already be passed in `args`.

Reply:
right. 

can you say anything on how to generalize the test and remove `_3270font` from https://github.com/NixOS/nixpkgs/pull/93533/files#diff-ed55ece2fe82ed6a0c2814164e107ae0R5

Reply:
You could make the `test.nix` return function instead of expression. Example:

https://github.com/NixOS/nixpkgs/blob/7adb91df59d6b024693c1278fdf5acfc498cbb40/pkgs/desktops/gnome-3/default.nix#L4
https://github.com/NixOS/nixpkgs/blob/7adb91df59d6b024693c1278fdf5acfc498cbb40/pkgs/desktops/gnome-3/update.nix#L1-L2